content/_layouts/developerbook.html.haml

```diff
@@ -56,7 +56,7 @@ layout: default
 
         %ul
           %li
-            %a{:href => 'http://reports.jenkins.io/reports/core-taglib/jelly-taglib-ref.html', :target => '_blank'}
```

Comment:
Is this supposed to work?

Reply:
https://github.com/jenkins-infra/core-taglibs-report-generator

Reply:
https://trusted.ci.jenkins.io:1443/job/core-taglibs-report-generator/70/console last success less than two days ago.

Reply:
We are using a very old version of azure-cli, I am validating that but I pretty sure it's just a bug in the version we are using

```
ERROR: 
The specified parent path does not exist.
script returned exit code 1
```

Reply:
Looks like it doesn't create the parent directory as needed?

Reply:
Yep @daniel-beck , the current command doesn't support that use case cfr (PR#70)[https://github.com/jenkins-infra/pipeline-library/pull/70]